python/pythonlab/neighborhood/tests/test_painter.py

```diff
@@ -1,4 +1,11 @@
 from neighborhood.painter import Painter
+from neighborhood.support.world import World
+from support.constants import SAMPLE_MAZE
+
+def setUp():
+  world = World()
+  # Set up the world to be from a string rather than trying to load a file.
+  world.set_grid_from_string(SAMPLE_MAZE)
```

Comment:
I imagine set_grid_from_string and set_grid_from_file come with World? Or we've defined them elsewhere?

Reply:
Yes - https://github.com/code-dot-org/code-dot-org/blob/c3ae88126cf6a64b5e891bff8646c3ac74751268/python/pythonlab/neighborhood/neighborhood/support/world.py#L20-L24

Non-blocking: similar question to [this comment](https://github.com/code-dot-org/code-dot-org/pull/63356#discussion_r1919269297) about `GridFactory` static methods...wondering about adding these to a `grid_helpers` module.

Reply:
I think I'll clean these up in my next pr just to keep things moving